Makefile

```diff
@@ -1,6 +1,6 @@
 PWD = $(shell pwd)
 
-all: lint build
+all: lint build preview
```

Comment:
Can we revert this? I never use preview, not really needed as a default make target